runtime/src/lib.rs

```diff
@@ -240,16 +240,9 @@ impl Contains<<Runtime as frame_system::Config>::RuntimeCall> for CallFilter {
             RuntimeCall::Content(content::Call::<Runtime>::update_channel_privilege_level {
                 ..
             }) => false,
-            RuntimeCall::Content(content::Call::<Runtime>::update_channel_nft_limit { .. }) => {
```

Comment:
Is it essential to enable `update_channel_nft_limit` for creator tokens to work?

Reply:
No, it is probably due to older modifications. 
I have it re-filtered again

Reply:
here https://github.com/Joystream/joystream/pull/4871/commits/ead0209600d711681888c6aff1ed222e07422b4e